packages/patternfly-react/src/components/SessionTimeout/SessionTimeout.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { SessionTimeout, SessionTimeoutModal } from './index';
+
+const noop = () => {};
+
+test('SessionTimeoutModal', () => {
+  const component = mount(
+    <SessionTimeoutModal logoutFnc={noop} continueFnc={noop} show>
+      <div>Session timeout ...</div>
+    </SessionTimeoutModal>
+  );
+  expect(component.render()).toMatchSnapshot();
+});
+
+test('SessionTimeout Dialog not displayed when timeLeft bigger than displayBefore', () => {
+  const component = mount(
+    <SessionTimeout
+      logoutFnc={noop}
+      continueFnc={noop}
+      displayBefore={10}
+      timeLeft={11}
+      units="seconds"
+    />
+  );
+
+  expect(component.render()).toMatchSnapshot();
```

Comment:
It would probably be better to do something like 
`expect(view.find(SessionTimeOutModal).prop('show').toBe(false)` it is more explicit than snapshots.